README.md

```diff
@@ -1,52 +1,54 @@
 
 ![](https://raw.githubusercontent.com/solidusio/solidus/master/solidus.png)
 
-* [solidus.io](http://solidus.io/)
-* [Join our Slack](http://slack.solidus.io/) ([solidusio.slack.com](http://solidusio.slack.com))
-* [solidus-security](https://groups.google.com/forum/#!forum/solidus-security) mailing list
+- [solidus.io](http://solidus.io/)
+- [Documentation](https://guides.solidus.io)
+- [Join our Slack](http://slack.solidus.io/) ([solidusio.slack.com](http://solidusio.slack.com))
+- [solidus-security](https://groups.google.com/forum/#!forum/solidus-security) mailing list
 
-Summary
--------
+## Summary
 
-Solidus is a complete open source e-commerce solution built with Ruby on Rails. It
-is a fork of Spree.
+Solidus is a complete open source ecommerce solution built with Ruby on Rails.
+It is a fork of [Spree](https://spreecommerce.org).
 
-Solidus actually consists of several different gems, each of which are maintained
-in a single repository and documented in a single set of
-[online documentation](http://docs.solidus.io/). By requiring the
-[`solidus`](https://github.com/solidusio/solidus) gem you automatically require all
-of the necessary gem dependencies which are:
+See the [Solidus class documentation](http://docs.solidus.io) and the [Solidus
+Guides](https://guides.solidus.io) for information about the functionality that
```

Comment:
I can only access the `http` version. Maybe it's better to use that link version?

Reply:
Thanks for pointing this out. Going to make sure that we get `https` today.